src/io/pfsgetstatus.c

```diff
@@ -71,8 +71,9 @@ void __osPfsGetOneChannelData(int channel, OSContStatus *data) {
     requestformat = *(__OSContRequesFormatShort *)ptr;
     data->errno = CHNL_ERR(requestformat);
 
-    if (data->errno == 0) {
+    if (data->errno) {
```

Comment:
```suggestion
    if (data->errno != 0) {
```
There is also another instance of this in `__osPfsGetInitData`.